core/src/main/java/ai/timefold/solver/core/impl/domain/variable/declarative/AffectedEntitiesUpdater.java

```diff
@@ -151,11 +149,11 @@ private boolean updateEntityShadowVariables(GraphNode<Solution_> entityVariable,
     }
 
     private boolean updateLoopedStatusOfEntity(Object entity, int entityId,
```

Comment:
This method can be made generic in <Entity_>; `Object` carries no semantics, `Entity_` makes it clear which `Object`s go together.